setup.bat

```diff
@@ -14,6 +14,20 @@ IF NOT EXIST pybind11-2.2.4 (
   git clone --branch v2.2.4 https://github.com/pybind/pybind11.git pybind11-2.2.4
 )
 
+IF NOT EXIST dlib-19.17 (
+  wget https://github.com/davisking/dlib/archive/v19.17.zip -O dlib-v19.17.zip --no-check-certificate
+  unzip dlib-v19.17.zip
+  PUSHD dlib-19.17
+  MKDIR Build
```

Comment:
Use lowercase for ```MKDIR```. Only built-in commands/keywords are upper case but ```mkdir``` is an actually a program.